src/zarate.jl

```diff
@@ -105,7 +105,7 @@ function ordering_problem(layout::Zarate, graph, layer2nodes)
     m = Model(layout.ordering_solver)
     set_silent(m)
 
-    T = JuMP.Containers.DenseAxisArray{VariableRef,1,Tuple{Vector{Int64}},Tuple{Dict{Int64,Int64}}}
+    T = JuMP.Containers.DenseAxisArray{VariableRef,1,Tuple{Vector{Int64}},Tuple{JuMP.Containers._AxisLookup{Dict{Int64,Int64}}}}
```

Comment:
This doesn't seem rightm, since it is accessing a function marked with an `_` which generally indicates private.

Reply:
Maybe we should just let `node_is_before` be a `Vector{Any}`
probably wouldn't hurt performance much.

Reply:
I am testing it, and I believe it would be much better so that we avoid any further truple if the JuMP implementation changes

Reply:
I tested the proposed change and it works fine; I also implemented it in my updated commit